commands/operator-sdk/cmd/build.go

```diff
@@ -206,21 +190,35 @@ func buildFunc(cmd *cobra.Command, args []string) {
 
 			s := &scaffold.Scaffold{}
 			err = s.Execute(cfg,
-				&scaffold.TestFrameworkDockerfile{},
+				&scaffold.TestFrameworkDockerfile{
+					Multistage: projutil.IsDockerMultistage(),
```

Comment:
For the test framework dockerfile, we require docker 17.05+ due to ARG-before-FROM. This means that the docker version is already new enough for multistage builds, so I don't think it makes sense to have a split multistage build for this image, unless we do something to not require ARG-before-FROM for the test image, which could be done with more scaffolding changes (hardcode the correct base image here before the build)

Reply:
I'll fix test framework Dockerfiles to multistage.